### DISCUSSION TITLE
Latex写论文投稿可行吗？

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#95] [灌水]
翔哥好，最近了解到overleaf写论文可以直接排版感觉起来方便一些，尤其是改投杂志修改格式实在太麻烦了。但是我看到很多杂志没有overleaf现成的模版，这样还可以使用吗？

**Comment by ShixiangWang** [comment]
具体要根据每个期刊提供的投稿指南看喔，绝大部分的期刊应该都是支持LaTex的。

**Reply by zhaoliang0302** [reply]
大部分是支持的，但是很多没有提供模版，自己造模版也不太现实

**Reply by ShixiangWang** [reply]
支持的一般提供模板，不支持的一般不提供。你可以仔细看看你一般想提稿的期刊有没有，如果能大部分覆盖的话应该是有价值的。

**Reply by zhaoliang0302** [reply]
你一般是直接用word吗？听听大神建议

**Reply by ShixiangWang** [reply]
嗯。我很少自己投稿，之前博士一般导师弄这个。也很少涉及公式什么的，没有用过latex投稿。
